### PR TITLE
Adding wasm32

### PR DESCRIPTION
**Pull request by huhlig**
You forgot wasm32!

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -97,6 +98,18 @@ pub trait Platform: Sized {
         }
     }
     #[inline(always)]
+    fn wasm32(self, _input: Self) -> Self {
+        #[cfg(target_arch="wasm32")]
+            {
```
Comment (Xaeroxe) [inline src/lib.rs:103]: Can you fix the whitespace?  Looks like you used tabs

**Review by Xaeroxe — CHANGES_REQUESTED** [review]

File: src/lib.rs (hunk 1 above)
Comment (huhlig) [reply src/lib.rs:103]: They are showing up as spaces... I'm confused.

**Comment by Moxinilian** [comment]
What if you use ``wasm32-unknown-emscripten``?

**Comment by Xaeroxe** [comment]
I guess it'd arbitrarily pick one based on the ordering you specify.

**Comment by Rhuagh** [comment]
It would pick the first matching you call yeah.

**Pull request merged**